run/sensitivity.py

```diff
@@ -1,139 +1,476 @@
 import pandas as pd
 from pathlib import Path
-import sys
 import argparse
+from collections import Counter
+
 
 def read_sensitivity(options=None):
 
-    if options is None or options.get('gw') is None:

```

Comment:
Is there any reason why this if condition is no longer here? Does it work fine whether it is wildcard or preseason?

Reply:
Since I added another command line argument (all_gws) I had to change the if options logic a bit. It still works fine for wildcard or preseason. The user can now use command line arguments like

python sensitivity.py --al_gws y
python sensitivity.py --gw 7 --wildcard n
python sensitivity.py  --gw 7 --wildcard y

or they can run without command line arguments and will then be asked first if they want to display a summary of buys and sells for all GWs. If they input n the script will continue like it did before